schemas/dependencies/dynatrace-environment-1.yml

```diff
@@ -15,8 +15,11 @@ properties:
     type: string
   environmentUrl:
     type: string
+  apiToken:
```

Comment:
@OliviaHY will this token be used for data ingestion or is it supposed to be an admin token that is only used to provision other tokens?

Reply:
More context in https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/76998

Reply:
This token will be the admin token. I have been naming it bootstrap token but this is a minor detail. More details about three different tokens in https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/76998#note_7281232

Reply:
Ok aligned the naming now to `bootstrapToken` :+1: